docs/data/joy/components/select/SelectMultipleAppearance.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import Select from '@mui/joy/Select';
+import Option from '@mui/joy/Option';
+import { Box, Chip } from '@mui/joy';
+
+export default function SelectMultipleAppearance() {
+  return (
+    <Select
+      multiple
+      defaultValue={['dog', 'cat']}
+      renderValue={(selected) => (
+        <Box sx={{ display: 'flex', gap: '0.25rem' }}>
+          {selected.map((selectedOption) => (
+            <Chip variant="outlined" color="success">
```

Comment:
```suggestion
            <Chip variant="soft" color="primary">
```

I think it looks better than `outlined` and `success`.

<img width="294" alt="image" src="https://github.com/mui/material-ui/assets/18292247/0f8873bd-724d-4af0-83b8-e6b26034e722">